src/saga/adaptors/pbspro/pbsprojob.py

```diff
@@ -851,11 +781,12 @@ def _job_get_info(self, job_id, reconnect):
 
         # run the PBS 'qstat' command to get some infos about our job
         # TODO: create a PBSPRO/TORQUE flag once
-        if 'PBSPro_1' in self._commands['qstat']['version']:
-            qstat_flag = '-fx'
-        else:
-            qstat_flag ='-f1'
-            
+
+        pbs_version = self._commands['qstat']['version'],
+        if   '14.2'     in pbs_version: qstat_flag = '-fx'  # Cheyenne
```

Comment:
pbs version is 18.2.<some sub version number> currently. We need to check for '18.2' instead of '14.2'